libcontainer/container_linux.go

```diff
@@ -376,6 +383,14 @@ func (c *linuxContainer) Checkpoint(criuOpts *CriuOpts) error {
 		}
 	}
 
+	// append optional manage cgroups mode
+	if criuOpts.ManageCgroupsMode != 0 {
+		if err := c.checkCriuVersion("1.6"); err != nil {
```

Comment:
The dash symbol in a criu version means that it was compiled from git. So if we found dash we can increment the last number in version.
For example:
1.6-smth should be converted into 1.7
1.6.2-smth should be converted int 1.6.3